mteb/benchmarks/benchmarks.py

```diff
@@ -556,7 +609,7 @@ def load_results(
 
 
 MTEB_multilingual = Benchmark(
-    name="MTEB(Multilingual)",
+    name="MTEB(Multilingual, beta)",
```

Comment:
Make sure to change the `default_benchmark` to this in the leaderboard.

Reply:
Fixed